microservices/password-manager/utils/charset-utils.ts

```diff
@@ -1,3 +1,5 @@
+import { CommandOutput } from "command-executor-lib/dist/command-output.js";
```

Comment:
Importing an enum that is not exported in the public interface is not a good idea.